_data/projects/pablo.yml

```diff
@@ -0,0 +1,10 @@
+name: Pablo
+desc: A Color Palette management app
+site: https://ajsaraujo.github.io/pablo/index.html
```

Comment:
Good for users, but how will someone trying to contribute will reach to the Github repo from there?

Reply:
Makes sense, I didn't thought of that.

Reply:
Cool